src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs

```diff
@@ -220,7 +220,7 @@ public static IResourceBuilder<RedisResource> WithRedisInsight(this IResourceBui
         }
         else
         {
-            containerName ??= $"{builder.Resource.Name}-insight";
+            containerName ??= "insight";
```

Comment:
To match scheme of Redis Command above?
```suggestion
            containerName ??= "redisinsight";
```

Reply:
Thanks, also improved a test since there was none for this one.